pkg/kubectl/cmd/get.go

```diff
@@ -110,19 +116,23 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 	if isWatch || isWatchOnly {
 		r := resource.NewBuilder(mapper, typer, f.ClientMapperForCommand()).
 			NamespaceParam(cmdNamespace).DefaultNamespace().AllNamespaces(allNamespaces).
+			FilenameParam(enforceNamespace, filenames...).
 			SelectorParam(selector).
 			ResourceTypeOrNameArgs(true, args...).
 			SingleResourceType().
+			Latest().
```

Comment:
If we don't using Latest() and using file here, the runtime.Object is decoded from the file, not from a server. so using Latest() here to get runtime.Object from a server. What's your advice?